web/geo/README.md

```diff
@@ -0,0 +1,12 @@
+# Overview
```

Comment:
Maybe we should add some of this to the wiki after it has been merged?

Reply:
Yes, I will create a complete wiki for how to edit. Added a checkbox to the PR